### PR TITLE
bugfix position toolbar on focus

### PR DESCRIPTION
**Pull request by Nihaux**
Bug reproduction:

1. select a word
2. click outside editor
3. click in the editor, not in a text

### CODE REVIEW

File: draft-js-inline-toolbar-plugin/src/components/Toolbar/index.js [hunk 1]
```diff
@@ -16,18 +16,22 @@ export default class Toolbar extends React.Component {
...
-      transform: 'translate(-50%) scale(1)',
-      transition: 'transform 0.15s cubic-bezier(.3,1.2,.2,1)',
-    } : {
-      transform: 'translate(-50%) scale(0)',
-    };
-    this.setState({
-      position,
-    });
+    // need to wait a tick for window.getSelection() to be accurate
+    // when focusing editor with allready present selection
```
Comment (nikgraf) [inline draft-js-inline-toolbar-plugin/src/components/Toolbar/index.js]: can you fix the typo? allready -> already

**Comment by nikgraf** [comment]
Looks good, thx!

**Comment by stevewillard** [comment]
Any chance we could get this merged?

**Comment by nikgraf** [comment]
@stevensacks done

@Nihaux thx for the fix!

**Pull request merged**